crowbar_framework/spec/models/service_object_spec.rb

```diff
@@ -107,7 +107,7 @@
         dns_proposal.elements["dns-client"] = ["admin"]
         dns_proposal.elements["dns-server"] = ["admin"]
 
-        dns_service.stubs(:role_constraints).returns({ "dns-client" => { "unique" => true, "admin" => true } })
+        allow(dns_service).to receive(:role_constraints).and_return({ "dns-client" => { "unique" => true, "admin" => true } })
```

Comment:
Metrics/LineLength: Line is too long. [126/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.